micro-core/src/main/java/com/aol/micro/server/spring/SpringContextFactory.java

```diff
@@ -47,16 +60,7 @@ public SpringContextFactory(Config config, Class c, Set<Class> classes){
 	}
 	
 	public SpringContextFactory(Config config, Set<Class> classes) {
```

Comment:
This constructor isn't used as far as I can tell, so it could be removed - along with the Null check in the main constructor.